lib/inv_gcr_quda.cpp

```diff
@@ -240,20 +255,25 @@ namespace quda {
     }
 
     // Check to see that we're not trying to invert on a zero-field source
-    if (b2 == 0) {
-      if (param.compute_null_vector == QUDA_COMPUTE_NULL_VECTOR_NO) {
+    if (param.compute_null_vector == QUDA_COMPUTE_NULL_VECTOR_NO) {
+      bool zero_src = true;
+      for (auto i = 0u; i < b.size(); i++) {
+        if (b2[i] == 0) {
+          warningQuda("inverting on zero-field source");
```

Comment:
Similar comment to above, printing the index of the source that's zero

Reply:
f0d9f3c8dc79bdb4ed751b34193262c517a6b287